src/main/java/com/minecolonies/coremod/client/gui/ConfigGUI.java

```diff
@@ -0,0 +1,51 @@
+package com.minecolonies.coremod.client.gui;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.minecolonies.coremod.configuration.ConfigurationHandler;
+import com.minecolonies.coremod.lib.Constants;
+
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.resources.I18n;
+
+import net.minecraftforge.common.config.ConfigElement;
+import net.minecraftforge.fml.client.config.DummyConfigElement;
+import net.minecraftforge.fml.client.config.GuiConfig;
+import net.minecraftforge.fml.client.config.IConfigElement;
+
+/**
+ * Represents the GUI Screen for the in-game GUI.
+ */
+public class ConfigGUI extends GuiConfig 
+{
+    public ConfigGUI(GuiScreen parentScreen) 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)